src/lib.rs

```diff
@@ -185,6 +187,28 @@ impl<T> Glog<T> {
         }
     }
 
+    /// Sets whether or not the span name is included. Defaults to true.
+    ///
+    /// If span names are not included, then the fields from all spans are
+    /// printed as a single list of fields.  This compacts the output and reduces
+    /// the complexity of the output's structure.
+    ///
+    /// # Example when `true`
+    /// ```no_compile
+    /// I0731 16:23:45.674465 990039 examples/tokio.rs:38] [parent_task{subtasks: 10, reason: "testing"}, subtask{number: 10}] polling subtask, number: 10
+    /// ```
+    ///
+    /// # Example when `false`
+    /// ```no_compile
+    /// I0731 16:23:45.674465 990039 examples/tokio.rs:38] [subtasks: 10, reason: "testing", number: 10] polling subtask, number: 10
+    /// ```
```

Comment:
Note that if you're using the gnome terminal, you can copy and paste output with HTML-formatted color codes:

```suggestion
    /// # Example Output
    /// With `with_span_names` set to true:
    /// <pre>
    /// I0731 16:23:45.674465 990039 examples/tokio.rs:38] [parent_task{subtasks: 10, reason: "testing"}, subtask{number: 10}] polling subtask, number: 10
    /// </pre>
    ///
    /// With `with_span_names` set to false:
    /// <pre>
    /// I0731 16:23:45.674465 990039 examples/tokio.rs:38] [subtasks: 10, reason: "testing", number: 10] polling subtask, number: 10
    /// <pre>
```